### PR TITLE
Watch later

### PR DESCRIPTION
**Pull request by atharvabhanage02**
This PR contains : 

- WatchLater Feature
- Like Feature
- Both the routes are protected
- User can either add video to Watch Later/Like from Explore Page or from the Single Video Page
 
Files to look for : 
- WatchLater Context : `src/Context/WatchLaterContext`
- LikeContext : `src/Context/LikeContext`
- Pages 
         - Watch Later Videos Page : `src/Pages/WatchLaterPage`
         - Like Videos Page : `src/Pages/LikePage`

Deploy Preview :  https://deploy-preview-4--cricplay.netlify.app/

### CODE REVIEW

File: src/Components/VideoMenu/VideoMenu.jsx [hunk 1]
```diff
@@ -10,25 +10,82 @@ import {
...
+              <div>Like</div>
+            </div>
+          )}
+          {pathname === "/playlist/:playlistId" && (
+            <div className="menu-icons">
+              <RiHistoryLine className="home-page-icons " />
+              <div>Remove from Playlist</div>
+            </div>
+          )}
+          {/* Playlist Modal will come here */}
```
Comment (aniketsingh13) [inline src/Components/VideoMenu/VideoMenu.jsx:86]: remove the commented code

**Review by aniketsingh13 — APPROVED** [review]
code structure is good
good naming convention 
good to merge

File: src/App.js [hunk 2]
```diff
@@ -4,13 +4,34 @@ import { Explore } from "./Pages/Explore/Explore";
 import { Login } from "./Pages/Login/Login";
 import { Signup } from "./Pages/Signup/Signup";
 import { SingleVideoPage } from "./Pages/SingleVideoPage/SingleVideoPage";
+import { WatchLaterPage } from "./Pages/Watch-Later-Page/WatchLaterPage";
```
Comment (sri-rishi) [inline src/App.js:7]: Suggestion: you can create index.js in pages folder and import files from there

File: src/Context/LikeContext/LikeContext.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  const [likedVideos, setLikedVideos] = useState([]);
+  const { auth } = useAuth();
+  useEffect(() => {
+    if (auth.isLogIn) {
+      (async () => {
+        try {
+          const response = await getLikedVideos(auth.tokenValue);
+          setLikedVideos(response.data.likes);
+        } catch (error) {
+          console.log("error occured while getting liked videos", error);
```
Comment (sri-rishi) [inline src/Context/LikeContext/LikeContext.js:16]: suggestion: for errors you can use `console.error` method

**Review by sri-rishi — APPROVED** [review]
Code looks good, you can merge

File: src/Context/LikeContext/LikeContext.js (hunk 3 above)
Comment (atharvabhanage02) [reply src/Context/LikeContext/LikeContext.js:16]: Thanks for suggesting that

**Pull request merged**